ooniprobe/Test/Test/WebConnectivity.mm

```diff
@@ -44,15 +45,17 @@ -(void)onEntry:(JsonResult*)json obj:(Measurement*)measurement{
 }
 
 -(int)getRuntime{
```

Comment:
I do not understand the logic for setting the `max_runtime`. Namely:

1. I'd check for the setting being disabled at the beginning and return `-1` immediately in such case

2. why is the case where `inputs != nil` different

3. why `self.settings.options.max_runtime` is (I assume) a copy of the settings stored into `WebConnectivity` rather than being just read from the settings

I feel like there is probably a good reason why that is the case, but the lack of documentation is making it more difficult for me to understand what we are doing in this diff.

Reply:
I will document it